core/templates/dev/head/pages/story-editor-page/navbar/story-editor-navbar.directive.html

```diff
@@ -71,8 +71,3 @@
     </ul>
   </li>
 </ul>
-<style>
-  story-editor-navbar .save-draft-button-padding {
```

Comment:
Why did you delete this? It's still used AFAICT.

Ditto elsewhere -- I see similar things in other files.

Reply:
Hi @seanlip, it's not needed it's handled through flex now.
this was used to move the save-draft text to the left when there was a change count

Reply:
But you've left the CSS class in the HTML tag. If you are deleting a CSS rule then make sure there are no more references left to the CSS class.

I saw similar things in other files, so please check.

Reply:
ah yes, i've removed it from codebase Thanks! and updated the class functions Thanks!